linode_api4/linode_client.py

```diff
@@ -733,6 +733,127 @@ def users(self, *filters):
+        """
+        Returns a  list of Payment Methods for this Account.
+        """
+
+        result = self.client.get(
+            "{}/payment-methods".format(Account.api_endpoint), model=self
+        )
+
+        return result["data"]
+
+    def add_payment_method(self, data, is_default, type):
+        """
+        Adds a Payment Method to your Account with the option to set it as the default method.
+        """
+
+        if type != "credit_card":
+            raise ValueError("Unknown Payment Method type: {}".format(type))
+
+        if (
+            "card_number" not in data["data"]
+            or "expiry_month" not in data["data"]
+            or "expiry_year" not in data["data"]
+            or "cvv" not in data["data"]
+            or not data["data"]
+        ):
+            raise ValueError("Invalid credit card info provided")
+
+        params = {"data": data, "type": type, "is_default": is_default}
+
+        self.client.post(
+            "{}/payment-methods".format(Account.api_endpoint),
+            model=self,
+            data=params,
+        )
+
+    def notification_list(self):
+        """
+        Returns a collection of Notification objects representing important, often time-sensitive items related to your Account.
+        """
+
+        result = self.client.get(
+            "{}/notifications".format(Account.api_endpoint), model=self
+        )
+
+        return result["data"]
+
+    def linode_managed_enable(self):
+        """
+        Enables Linode Managed for the entire account and sends a welcome email to the account’s associated email address.
+        """
+
+        self.client.post(
+            "{}/settings/managed-enable".format(Account.api_endpoint),
+            model=self,
+        )
+
+    def add_promo_code(self, promo_code):
+        """
+        Adds an expiring Promo Credit to your account.
+        """
+
+        params = {
+            "promo_code": promo_code,
+        }
+
+        self.client.post(
+            "{}/promo-codes".format(Account.api_endpoint),
+            model=self,
+            data=params,
+        )
+
+    def service_transfers_list(self):
+        """
+        Returns a collection of all created and accepted Service Transfers for this account, regardless of the user that created or accepted the transfer.
+        """
+
+        result = self.client.get(
+            "{}/service-transfers".format(Account.api_endpoint), model=self
+        )
+
+        return result["data"]
+
+    def service_transfer_create(self, entities):
+        """
+        Creates a transfer request for the specified services.
+        """
+
+        if not entities["entities"] or "linodes" not in entities["entities"]:
+            raise ValueError("Invalid entities provided: {}".format(entities))
+
+        params = {"entities": entities}
+
+        self.client.post(
+            "{}/service-transfers".format(Account.api_endpoint),
+            model=self,
+            data=params,
+        )
+
```

Comment:
We might want to keep list endpoint method names consistent with other methods in this file:
https://github.com/linode/linode_api4-python/blob/dev/linode_api4/linode_client.py#L58

e.g. `service_transfers_list` -> `service_transfers`

It also looks like some of these endpoints are returning dicts rather than the corresponding object type. For most list endpoints we can work around this using this automatically using the `_get_and_filter(...)` helper 👍